diff --git a/external/apache-http/Android.mk b/external/apache-http/Android.mk
index c348fdc..e0f3cf2 100644
--- a/Android.mk
+++ b/Android.mk
@@ -65,7 +65,7 @@ apache_http_packages := $(strip \
 include $(CLEAR_VARS)
 LOCAL_MODULE := org.apache.http.legacy.boot
 LOCAL_MODULE_TAGS := optional
-LOCAL_JAVA_LIBRARIES := $(apache_http_java_libs)
+LOCAL_JAVA_LIBRARIES := $(apache_http_java_libs) okhttp bouncycastle framework
 LOCAL_SRC_FILES := $(apache_http_src_files)
 LOCAL_SDK_VERSION := 21
 LOCAL_MODULE_TAGS := optional
